src/background/api/txHelper.js

```diff
@@ -299,15 +299,17 @@ export async function buildParatimeTxBody(params, wrapper) {
         oasis.quantity.fromBigInt(amount),
         oasisRT.token.NATIVE_DENOMINATION
     ]);
-    let feeAmount  = params.feeAmount||0
+    // Fee amount idiosyncrasy: UI presents it in 1e-9 always, regardless of paratime setting.
+    let feeAmountDecimal = new BigNumber(10).pow(cointypes.decimals)
+    let feeAmount  = new BigNumber(params.feeAmount||0).multipliedBy(decimal).dividedBy(feeAmountDecimal).toFixed()
```

Comment:
```suggestion
    let feeAmount  = new BigNumber(params.feeAmount ?? 0).multipliedBy(decimal).dividedBy(feeAmountDecimal).toFixed()
```

Reply:
See https://github.com/oasisprotocol/oasis-wallet-ext/pull/228#discussion_r789366590.

Reply:
stet. meant to handle `""`